client/lib/generate-password/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
+import { random, sample } from 'lodash';
+
+export const LETTER_CHARSETS = {
+	lowerChars: 'abcdefghjkmnpqrstuvwxyz'.split( '' ),
+	upperChars: 'ABCDEFGHJKMNPQRSTUVWXYZ'.split( '' ),
+};
+
+export const CHARSETS = {
+	...LETTER_CHARSETS,
+	digitChars: '23456789'.split( '' ),
```

Comment:
I've noticed we previously didn't include `1` as a possible character in our password generator. Any idea why? Should we include it if that was a mistake?

Reply:
I don't know the rationale in our specific case, but I've seen other password generators omit `0` and `1` to improve legibility. In many fonts, those characters are difficult to distinguish from `O` and `l`.

Reply:
> In many fonts, those characters are difficult to distinguish from `O` and `l`

Yup, I think that's the reason behind omitting a few characters.

Reply:
I mean it's at least debatable to whether we want to keep this behavior. Another strategy would be to use a monospaced font for the password field where those characters are usually easier distinguishable.

fwiw the first few characters are always the same structure and every fourth character is a letter which (imho) reduces the general "strongness" of the password and and is probably not good practice in terms of security. Also `Math.random` is not cryptographically secure which is also not ideal. My guess is for our use case it's fair enough though.

Reply:
What a weird rationale IMHO 😉  Especially considering that passwords are often just asterisks. 

What if I'm using a font where the A looks like a 4? 😆 

But happy to keep it as-is.